### PR TITLE
feat(tabs): fixed tabs

### LINKED ISSUE #1353: [Tabs] Allow option to stretch tab labels across the header
#### Bug, feature request, or proposal:

Feature Request: Add support for [fixed tab layout ](https://material.google.com/components/tabs.html#tabs-types-of-tabs)
#### What is the expected behavior?

Fixed tabs will fill up the tab group equally sized.
#### What is the current behavior?

Tabs cannot be marked as fixed.

### PR DESCRIPTION
**Pull request by dasois**
Fixed tabs grow until they fill the tab group.

Closes #1353

### CODE REVIEW

**Comment by andrewseguin** [comment]
This would be a welcome addition to tabs, but I'm trying to figure out if this is the right API. Fixed tabs to me seems to imply that the number of tabs cannot be changed dynamically, which is something we would like to support.

I'm leaning towards the material 1 direction of this feature and having it called md-stretch-tabs which I think is more expressive about its intent. However, I'm not yet convinced that it needs to be as complex (auto, always, never).

**Comment by andrewseguin** [comment]
Can you update the flag to be md-stretch-tabs?

**Comment by dasois** [comment]
done

**Comment by dasois** [comment]
updated demo app

File: src/demo-app/tabs/tab-group-demo.html [hunk 1]
```diff
@@ -27,3 +27,17 @@
     </template>
   </md-tab>
 </md-tab-group>
+
+<h1>Stretched Tabs</h1>
+<md-tab-group class="demo-tab-group" md-stretch-tabs>
+  <md-tab *ngFor="let tab of tabs; let i = index" [disabled]="i == 1">
+    <template md-tab-label>{{tab.label}}</template>
+    <template md-tab-content>
```
Comment (andrewseguin) [inline src/demo-app/tabs/tab-group-demo.html]: md-tab-content no longer exists. Try rebasing and testing it out in the demo app

**Comment by dasois** [comment]
sorry, had some issues rebasing.
New pull-request is: #1909

**Pull request closed without merging**